charts/temporal/templates/web-deployment.yaml

```diff
@@ -3,6 +3,10 @@ apiVersion: apps/v1
 kind: Deployment
 metadata:
   name: {{ include "temporal.componentname" (list $ "web") }}
+  {{- if or $.Values.additionalAnnotations $.Values.admintools.deploymentAnnotations }}
```

Comment:
```suggestion
  {{- if or $.Values.additionalAnnotations $.Values.web.deploymentAnnotations }}
```

Reply:
On seconds thoughts lets remove the gating.

Reply:
Removed the gating.